.github/workflows/insiders-fast.yml

```diff
@@ -18,9 +18,9 @@ jobs:
         run: |
           # If the last commit was greater than 24 hours ago, skip the build
           if [[ $(git log -1 --format=%ct) -lt $(date -d '24 hours ago' +%s) ]]; then
-            echo new=true >> $GITHUB_OUTPUT
-          else
             echo new=false >> $GITHUB_OUTPUT
+          else
```

Comment:
delta looks identical, does the script not execute properly because of the formatting?

Reply:
This switches the `new=true` and `new=false` output for this step.